Dockerfile

```diff
@@ -1,15 +1,20 @@
+# For more information, please refer to https://aka.ms/vscode-docker-python
 FROM python:3.9
 
 RUN mkdir /app
 WORKDIR /app
 
-# Install dependencies for pdf2image and tesseract
+RUN apt-get clean
```

Comment:
Probably best cleaning after the update:
>       clean
           clean clears out the local repository of retrieved package files. It removes everything but the lock file from /var/cache/apt/archives/ and
           /var/cache/apt/archives/partial/.